client/src/pages/Details.js

```diff
@@ -26,19 +24,16 @@ function Details() {
     : data.data.filter((data) =>
         data.title.toLowerCase().includes(search.toLowerCase())
       );
-  // function deleteFood(title) {
-  //   const newData = data.data.filter((data) => data.title !== title);
-  //   console.log(newData);
-  // }
+  console.log(data);
 
-  function deleteRow(id, e) {
-    axios.delete(`http://localhost:5000/food/${id}`).then((res) => {
-      console.log(res);
-      console.log(res.data);
-      const newFood = data.data.filter((data) => data.id !== id);
-      console.log(newFood);
-    });
-  }
+  const handleDelete = async (id) => {
+    try {
+      const foodItems = await axios.delete("http://localhost:5000/food/" + id);
+      console.log(foodItems);
```

Comment:
Can this be deleted?

```suggestion
      console.log(foodItems);
```